mainapp/templates/mainapp/request_form.html

```diff
@@ -41,51 +63,38 @@ <h3 id="myModalLabel">Allow Location - നിങ്ങളുടെ സ്ഥല
         <button class="btn" data-dismiss="modal" id="modal_ok_button" aria-hidden="true">Ok</button>
       </div>
     </div>
-  </div>
-
+	</div>
   <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.9.1/jquery.min.js"></script>
+  <script src="https://cdn.jsdelivr.net/npm/lodash@4.17.10/lodash.min.js" integrity="sha256-/GKyJ0BQJD8c8UYgf7ziBrs/QgcikS7Fv/SaArgBcEI=" crossorigin="anonymous"></script>
```

Comment:
Not sure where lodash is getting used.

Reply:
I am not using it.

Reply:
Please remove unwanted dependencies if you are not  it